ASTNode.h

```diff
@@ -18,24 +18,29 @@ class ASTNode {
         NODE_COMPREHENSION, NODE_LOADBUILDCLASS, NODE_AWAITABLE,
         NODE_FORMATTEDVALUE, NODE_JOINEDSTR, NODE_CONST_MAP,
         NODE_ANNOTATED_VAR, NODE_CHAINSTORE, NODE_TERNARY,
-        NODE_KW_NAMES_MAP,
+        NODE_KW_NAMES_MAP, NODE_CALL_INTRINSIC_1, NODE_CALL_INTRINSIC_2,
 
         // Empty node types
         NODE_LOCALS,
     };
 
-    ASTNode(int type = NODE_INVALID) : m_refs(), m_type(type), m_processed() { }
+    ASTNode(int type = NODE_INVALID, bool unpacked = false) : m_refs(), m_type(type), m_processed(), m_unpacked(unpacked) { }
     virtual ~ASTNode() { }
 
     int type() const { return internalGetType(this); }
 
     bool processed() const { return m_processed; }
     void setProcessed() { m_processed = true; }
 
+    bool unpacked() const { return m_unpacked; }
+    void setUnpacked() { m_unpacked = true; }
```

Comment:
It seems these aren't used -- only the unpacked property on PycRef is used in your changes below...